cmd/zed/lake/command.go

```diff
@@ -77,7 +76,7 @@ func ParseOrder(s string) (zbuf.Order, error) {
 func ParseIDs(args []string) ([]ksuid.KSUID, error) {
 	ids := make([]ksuid.KSUID, 0, len(args))
 	for _, s := range args {
-		id, err := ParseID(s)
+		id, err := lake.ParseID(s)
```

Comment:
This doesn't belong in lake as it has no dependencies on lake and lake does not depend on it.   I think it goes in the forthcoming package that will implements the common interface between zed lake and zed api.

Reply:
Cool. Can we leave this here until that forthcoming package is established?